requirements.txt

```diff
@@ -29,3 +29,4 @@ aliyun-python-sdk-ecs==4.24.25
 arcaflow-plugin-sdk
 wheel
 git+https://github.com/vmware/vsphere-automation-sdk-python.git@v8.0.0.0
+git+https://github.com/arcalot/arcaflow-plugin-kill-pod.git@main
```

Comment:
Is this going to stay as a separate repo or is it going to be merged into https://github.com/arcalot/arcaflow-plugin-sdk-python in the future?

Reply:
correct, it will be one plugin per repo